src/Security/PostVoter.php

```diff
@@ -23,6 +23,8 @@
  * See https://symfony.com/doc/current/security/voters.html
  *
  * @author Yonel Ceruto <yonelceruto@gmail.com>
+ *
+ * @extends Voter<non-empty-string, Post>
```

Comment:
Also see https://github.com/phpstan/phpstan/issues/10573

Using `non-empty-string` to avoid 

```
 ------ ------------------------------------------------------------------------------------- 
  Line   src/Security/PostVoter.php                                                           
 ------ ------------------------------------------------------------------------------------- 
  37     Asserted type string for $attribute with type string does not narrow down the type.  
 ------ ------------------------------------------------------------------------------------- 
 ```